### PR TITLE
Symmetric V2 support added

### PR DESCRIPTION
**Pull request by 0xCoDeFi**
##### Short Description (to be shown on DefiLlama): This is an update for existing listing for Symmetric. The new change is for Symmetric V2 and xDai renamed to Gnosis.

### CODE REVIEW

**Comment by g1nt0ki** [comment]
@0xCoDeFi there seems to be merge conflict, can you please rebase code against latest commit in `master` branch to fix this

**Comment by llamatester** [comment]
Error while running adapter at projects/symmetric: 
        
 
  
    
    
    Error: 
        Unknown chain(s): gnosis
        Note: if you think that the chain is correct but missing from our list, please add it to 'projects/helper/chains.json' file

**Comment by 0xCoDeFi** [comment]
Resolved conflict. Thanks.

File: projects/symmetric/index.js [hunk 1]
```diff
@@ -19,28 +18,42 @@ async function getTVL(subgraphName, block) {
...
-  methodology: `Symmetric is an Automated Market Maker (AMM) and a Decentralized Exchange (DEX), running on the Celo and xDai networks.
-  Symmetric TVL is pulled from the Symmetric subgraph and includes deposits made to Symmetric xDai and Symmetric Celo liquidity pools.`,
+  methodology: `Symmetric is an Automated Market Maker (AMM) and a Decentralized Exchange (DEX), running on the Celo and Gnosis networks.
+  Symmetric TVL is pulled from the Symmetric subgraph and includes deposits made to Symmetric Gnosis and Symmetric Celo V1 & V2 liquidity pools.`,
   celo:{
     tvl: celo
   },
-  xdai:{
-    tvl: xdai
+  gnosis:{
```
Comment (g1nt0ki) [inline projects/symmetric/index.js:55]: can you change the name here to `xdai`

File: projects/symmetric/index.js [hunk 2]
```diff
@@ -19,28 +18,42 @@ async function getTVL(subgraphName, block) {
...
+    tvl: gnosis
   },
+  tvl: sdk.util.sumChainTvls([gnosis, celo])
```
Comment (g1nt0ki) [inline projects/symmetric/index.js:58]: this line can be removed, we no longer need this.

**Review by g1nt0ki — CHANGES_REQUESTED** [review]

File: projects/symmetric/index.js (hunk 1 above)
Comment (0xCoDeFi) [reply projects/symmetric/index.js:55]: Sure

File: projects/symmetric/index.js (hunk 2 above)
Comment (0xCoDeFi) [reply projects/symmetric/index.js]: Removed.

**Comment by llamatester** [comment]
The adapter at projects/symmetric exports TVL: 
        
 
 
    celo                      343.58 k
    xdai                      2.63 M
    
    total                    2.97 M

**Review by 0xCoDeFi — COMMENTED** [review]
Please let me know if you have any more comments. Thanks for your help.

Best,

**Pull request merged**